src/gui/properties/propertieswidget.h

```diff
@@ -55,15 +55,18 @@ class QAction;
 class QTimer;
 QT_END_NAMESPACE
 
-class PropertiesWidget: public QWidget, private Ui::PropertiesWidget
+class PropertiesWidget : public QWidget, private Ui::PropertiesWidget
```

Comment:
Although i prefer it with the extra space if you don't intend to change it in uncrustify i'll undo it for consistency.